src/control/cmd/dmg/system.go

```diff
@@ -195,7 +226,7 @@ func (cmd *systemQueryCmd) Execute(_ []string) error {
 		return nil
 	}
 
-	if len(cmd.Ranks) == 1 {
+	if len(cmd.RankList) == 1 {
```

Comment:
This is a bug, right? I've made this mistake as well. `cmd.RankList` is a string, not a list, so this is testing that it's a one-character string which will never be true. Hence my proposal that we only use the `List` form of these when we're actually dealing with slices.